README.md

```diff
@@ -335,17 +335,17 @@ est_py, est_nm, est_inv, confident_joint = estimate_py_and_noise_matrices_from_p
 The joint probability distribution of noisy and true labels, *P(s,y)*, completely characterizes label noise with a class-conditional *m x m* matrix.
 
 ``` python
-from cleanlab.latent_estimation import estimate_joint
+from cleanlab.count import estimate_joint
 joint = estimate_joint(
-    s=noisy_labels,
+    labels=noisy_labels,
     psx=probabilities,
     confident_joint=None,  # Provide if you have it already
 )
 ```
 
 ## The Polyplex
 
-The key to learning in the presence of label errors is estimating the joint distribution between the actual, hidden labels ‘*y*’ and the observed, noisy labels ‘*s*’. Using `cleanlab` and the theory of confident learning, we can completely characterize the trace of the latent joint distribution, *trace(P(s,y))*, given *p(y)*, for any fraction of label errors, i.e. for any trace of the noisy channel, *trace(P(s|y))*.
+The key to learning in the presence of label issues is estimating the joint distribution between the actual, hidden labels ‘*y*’ and the observed, noisy labels ‘*s*’. Using `cleanlab` and the theory of confident learning, we can completely characterize the trace of the latent joint distribution, *trace(P(s,y))*, given *p(y)*, for any fraction of label errors, i.e. for any trace of the noisy channel, *trace(P(s|y))*.
 
 You can check out how to do this yourself here:
 1. [Drawing Polyplices](https://github.com/cleanlab/examples/blob/master/drawing_polyplices.ipynb)
```

Comment:
Move polyplex tutorial/discussion to confidentlearning-reproduce repo?  Or into internal readme page, eg. in the `utils/` folder?

Reply:
just removed